backend/app/Controllers/Admin/SpellsController.php

```
@@ -0,0 +1,514 @@
+
+		return ApiResponse::success([
+			'spells' => $spells,
+			'pagination' => [
+				'page' => $page,
+				'limit' => $limit,
+				'total' => $total,
+			],
+		], 'Spells fetched successfully', 200);
+	}
+
+	public function show(Request $request, int $id): Response
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)